security/responding.md

```diff
@@ -0,0 +1,143 @@
+   Threat Model, the discloser is notified and the process ends.
+   (Note that in all cases it is important to inform the reporter that
+   the disclosure has been seen and is being addressed as quickly as possible).
+1. Otherwise, the Fix Lead creates a private branch for development of the Fix,
+   and a Github Security Advisory to coordinate the disclosure.
+1. The Fix Lead responds to the reporter to let them know the work has begun to
+   address the vulnerability. **If appropriate, and if the reporter is willing,
+   the reporter is added to the Security Advisory and can help to develop the
+   fix. The reporter may also wish to help with writing the CVE entry. They
+   should in all cases be asked if they would like credit for identifying the
+   vulnerability in the CVE and disclosure**.
+1. The Fix Team urgently develops a fix or mitigation so that a realistic
+   timeline can be communicated to users.
+1. If the CVSS score is below 4.0, the Fix Team may - after appropriate
+   discussion and based on developer bandwidth, public holidays etc - decide to
+   slow down the release process.
+1. The fact that a vulnerability has been disclosed and a fix will be released
+   is communicated by the Fix Lead (working with the Fix Team) to the
+   knative-security-announce mailing list with an exact date and time planned
+   for the disclosure. If the optional embargoed disclosure step will be
+   followed (see next section), this date must be three weeks from when the
+   disclosure is released to the embargo list.
+1. This email should be actionable: it should be clear when users
+   should block time to apply patches, what migiations are available (if
+   appropriate), etc.
+
+### Optional, Within 1-14 Days of Disclosure (Private, Embargoed Disclosure)
+
+1. If, in the judgement of the Fix Lead, a private disclosure is warranted, the
+   Embargo List is emailed with details of the fix (including any proposed
+   patch/mitigation and actionable instructions) and the date it will be
+   publicly disclosed and released. 
+1. If this is the case, the Public Disclosure should not be
+   released until at least three weeks after this notification is sent, to give
+   vendors time to prepare to release the patch. In exceptional circumstances
+   this may be extended to four weeks to account for significant public
+   holidays etc.
+1. Note: This step is optional and should be reserved for significant
+   vulnerabilities that warrant this treatment. In general these are
+   vulnerabilities that could lead to remotely exploitable or privilege
+   escalation issues.
+1. In the unlikely event that a member of the PST list breaks embargo, or if
+   the exploit otherwise becomes public early, the PST Lead will decide whether
+   to accelerate the schedule for public disclosure and release of the fix.
+   When in doubt push forward and go public ASAP.
+
+### Within 1-21 Days of Disclosure (Fix Release Day)
```

Comment:
21 days seems inconsistent with notifying the embargo list on day 14, and waiting another 3 weeks for public disclosure.

Reply:
ha! Good point. The lower bound can be achieved when the embargo step is skipped (for non-remotely-explotable/low severity vulnerabilities etc), but the upper bound is clearly longer than 21 days here. Will fix.